Environment_Engine/Query/FloorGeometry.cs

```diff
@@ -63,7 +63,14 @@ public static Polyline FloorGeometry(this List<Panel> panelsAsSpace, double tole
 
             if (floorGeometry.Count < 1) return null;
 
-            return floorGeometry[0];
+                Polyline maxFloorGeometry = floorGeometry[0];
```

Comment:
Fix the indenting on this, this block is indented one too far.

Reply:
done